source/components/tables/tbutils.c

```diff
@@ -257,7 +257,7 @@ AcpiTbCheckDsdtHeader (
  *
  * FUNCTION:    AcpiTbCopyDsdt
  *
- * PARAMETERS:  TableDesc           - Installed table to copy
+ * PARAMETERS:  TableIndex         - Index of installed table to copy
```

Comment:
Also I can see problems in the patch description:
1. Should the body be empty? It's OK of being empty in ACPICA upstream, but not OK for linux upstream IMO.
2. There shouldn't be empty line between SOBs.